### ISSUE TITLE
Updater Setup Blocked By Administrator

### ISSUE THREAD

**Issue by Fffsteak** [Kwoth/NadekoBot#2344]
I've trying to install Nadeko for the first time, and I'm currently following the install guide. However when I attempt to install the bot updater I keep getting a Windows error message stating that the application is blocked by the admin for potential security risk. This keeps occurring even when I run the setup.exe as an admin, and have all firewalls and windows security features disabled. I'm currently running Windows 10. Any idea why this message is popping up?

**Comment by xnaas** [comment]
You mean like this prompt?

![img](https://xnaas.info/img/BoringMatureAfricanrockpython.png)

Or am I misunderstanding?

**Comment by Fffsteak** [comment]
@xnaas I'm already on solution 9 in that list with no change whatsoever.

**Comment by xnaas** [comment]
Sounds like something might be FUBAR with your Windows install then? If you've already tried the editing the registry keys, Group Policy changes, making sure the file is unblocked, and rebooting, I'm not really sure what to tell ya.

**Comment by Fffsteak** [comment]
orz I don't have the copy of windows to reinstall from, nor am I willing to go through that big ordeal. Shit man. I just wanted a nice bot.

**Comment by xnaas** [comment]
Did `sfc /scannow` return saying there is corruption it could not fix?

**Comment by Fffsteak** [comment]
Nope. No integrity issues found.

**Comment by Fffsteak** [comment]
What about this file is tripping the security? Because I've never had trouble like this downloading and installing .exes, even from less secure places.

**Comment by xnaas** [comment]
Dunno. All I know is when I go to install, I get the same window as you except I can click `Install`:

![img](https://xnaas.info/img/OldfashionedDifferentDogwoodtwigborer.png)

I assume you got the download link from here https://nadekobot.readthedocs.io/en/latest/guides/Windows%20Guide/ which takes you to here https://download.nadekobot.me ?

**Comment by Fffsteak** [comment]
Yup, that's where I'm downloading it from. Is there some hidden option in window's defender that I'm not disabling?

**Comment by xnaas** [comment]
Unfortunately, I'm not sure. I would have assumed *something* in that article would have covered your specific scenario.

Maybe someone will hop along that ran into the same issue. You might try digging through some of the older issues and see if this has come up before too.

**Comment by Fffsteak** [comment]
@xnaas I FIGURED IT OUT. I followed this on a crap shoot and now it's allowing me to install the updater.

https://superuser.com/questions/1252575/unable-to-install-clickonce-application-due-to-security-settings-windows-10

**Comment by xnaas** [comment]
Glad to hear. 

You should go ahead and close out this issue if it's fully resolved for you now. 😉
